### DISCUSSION TITLE
Adding custom fields to existing entities using medusa server application

### DISCUSSION THREAD

**Discussion by tarsusi** [medusajs/medusa#3134] [Q&A]
Hello,

I have created an question in stackoverflow, you can find the details [here](https://stackoverflow.com/questions/75269182/how-to-add-a-custom-field-to-existing-entity-in-medusajs).

I am wondering that how is it possible to extend existing entities.

**Comment by olivermrbl** [comment]
Answered your question in SO. You can follow along in our process of adding a Custom Fields API [here](https://github.com/medusajs/medusa/discussions/1947).
